### DISCUSSION TITLE
Section 1: NFT Exercise

### DISCUSSION THREAD

**Discussion by 7gbice** [Cyfrin/security-and-auditing-full-course-s23#60] [Q&A] [answered]
# At the end of the Course Section 1: Review, the Exercise given on
...
Section 1 NFT Challenge 👀
[Refresher NFT (Arb)](https://arbiscan.io/address/0x7a0f40757f6ba868b44ce959a1d4b8bc22c21d59)
[Refresher NFT (Sepolia)](https://sepolia.etherscan.io/address/0x76d2403b80591d5f6af2b468bc14205fa5452ac0)
...
What am i required of?

**Comment by cromewar** [comment]
Hello @7gbice 
Can you please be more specific with your question? please check [how to ask question](https://github.com/Cyfrin/security-and-auditing-full-course-s23/blob/main/how-to-ask-a-question.md), this way we can provide better support.

**Comment by cromewar** [comment, marked as answer]
We are closing this due to a lack of follow-up.
